### DISCUSSION TITLE
How to specify observation noise for FixedNoiseGP

### DISCUSSION THREAD

**Discussion by EvanClaes** [meta-pytorch/botorch#1473] [Q&A]
Hi,

I'm wondering what is the best way to specify observation noise when using a FixedNoiseGP together with NEI. The case I am working on has a fixed amount of observation noise calculated as a percentage from the true (noiseless) observation. Thus for each observation I know the exact amount of noise.

However for fitting the FixedNoiseGP the observations are standardized. The value with which they are scaled varies from iteration to iteration as more queries are evaluated, and thus the observation noiselevels (which are also scaled) vary as well.

Is this harmful for NEI? Would I be better off choosing a fixed scaling value for the standardization so that the specified noise levels for FixedNoiseGP are fixed as well?

**Comment by saitcakmak** [comment]
Hi @EvanClaes. I don't think it should matter as long as you're using the same scaling factors for the observations and the observation noise (within an iteration). We often just use the `Standardize` transform to standardize both the observations and the noise. The scaling factors `Standardize` uses are learned from the data and change from iteration to iteration.
